.gitignore

```diff
@@ -1 +1,5 @@
 install_pgks.sh
+*.py
```

Comment:
That's good to consider. The reason I do it this way, is because my development environment uses specially formatted python files instead of ipynb files. Instead, I can convert these files to ipynb files, and then convert these ipynb files to python files that don't get ignored inside nbconverted folders.